DependencyInjection/Compiler/ThemeCompilerPass.php

```diff
@@ -28,7 +28,7 @@ public function process(ContainerBuilder $container)
                 ->replaceArgument(2, null);
         }
 
-        $twigFilesystemLoaderDefinition = $container->getDefinition('twig.loader.filesystem');
+        $twigFilesystemLoaderDefinition = $container->getDefinition('twig.loader.native_filesystem');
```

Comment:
though `getDefinition()` may indeed need to be replaced with `findDefinition()` here